lib/health_cards/health_card.rb

```diff
@@ -98,8 +143,22 @@ def types
       def supports_type?(*type)
         !types.intersection(type).empty?
       end
+
+      protected
+
+      def parent_allowables(base = {})
+        self < HealthCards::HealthCard ? superclass.parent_allowables(base.merge(superclass.allowable)) : base
```

Comment:
`superclass.allowable` should already contain `superclass.parent_allowables`

```suggestion
        self < HealthCards::HealthCard ? base.merge(superclass.allowable) : base
```